File: CSamples/Chapter02/SumOfArray.c
#include <stdio.h>
#define LENGTH 10

int main() {
  int a[] = { 72, 68, 92, 88, 41, 53, 97, 84, 39, 55 };
  int sum, i;

  sum = 0;
  for (i = 0; i < LENGTH; i++) {
    sum += a[i];
  }
  printf("sum = %d\n", sum);

  return 0;
}

File: JavaSamples/Chapter02/SumOfArray.java
public class SumOfArray {
  public static void main(String[] args) {
    int[] a = { 72, 68, 92, 88, 41, 53, 97, 84, 39, 55 };
    int sum, i;
    sum = 0;

    for (i = 0; i < a.length; i++) {
      sum += a[i];
    }
    System.out.printf("sum = %d\n", sum);
  }
}